src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1735,37 +1695,20 @@ export function getChartOptions(
         colorStrategy,
     );
 
-    let drillableSeries = getDrillableSeries(
+    const drillableSeries = getDrillableSeries(
         seriesWithoutDrillability,
         drillableItems,
-        viewByAttribute,
+        [viewByAttribute, viewByParentAttribute],
         stackByAttribute,
         executionResponse,
         afm,
         type,
     );
 
-    let viewByParentAttribute: IUnwrappedAttributeHeadersWithItems;
-    if (isViewByTwoAttributes) {
-        viewByTwoAttributes = {
-            parent: attributeHeaderItems[VIEW_BY_DIMENSION_INDEX][PARENT_ATTRIBUTE_INDEX],
-            children: attributeHeaderItems[VIEW_BY_DIMENSION_INDEX][PRIMARY_ATTRIBUTE_INDEX],
-        };
-
-        viewByParentAttribute = findAttributeInDimension(
-            dimensions[VIEW_BY_DIMENSION_INDEX],
-            attributeHeaderItems[VIEW_BY_DIMENSION_INDEX],
-            PARENT_ATTRIBUTE_INDEX,
-        );
-        if (viewByParentAttribute) {
-            drillableSeries = getDrillableSeriesWithParentAttribute(drillableSeries, viewByParentAttribute);
-        }
-    }
-
     const series = assignYAxes(drillableSeries, yAxes);
 
-    let categories = viewByTwoAttributes
-        ? getCategoriesForTwoAttributes(viewByTwoAttributes)
+    let categories = viewByParentAttribute
```

Comment:
Please add test case for this

Reply:
Do you mean to assert `categories` which is returned from `getChartOptions`?

Reply:
Fixed :)